content/blog/a-summary-of-the-july-retrospectives/index.md

```diff
@@ -0,0 +1,98 @@
+
+For the original GA tags for July, you can find our 'scorecard' on how we did.  64.3% were released within the targeted timeframes, 37.7% were not released within targets.  Of the ones that were not released within targets, 7 of them missed the targets by more than 5 days, which I will retrospectively refer to as "the Sickest Seven".  For a detailed breakdown of these targets per platform and version, see [adoptium/issues/157](https://github.com/adoptium/adoptium/issues/157).  
+
+## The Sickest Seven
+As [issue 157](https://github.com/adoptium/adoptium/issues/157) describes, 7 of the 42 products released badly missed the target goals (by 5 or more days).  
+
+Let's look at the Sickest Seven, and determine next actions for improvement.  
+From the initial July release activity, the _Sickest Seven_ (7 products that took more than 5 business days over the 2 or 7 day target to publish), plus the _two outliers_ from the respins:
+
+| Platform | Version | Days over Target | Underlying Reasons / Notes | Actions to Improve | 
+| -- | -- | -- | -- | -- |
+| x64 Mac | JDK 8 | 14 | Owners need to treat as urgent, onboarding difficulties, slow connectivity | Clarify requirements with owners and improve access to TCK machines |
+| x64 Mac | JDK 18 | 27 | Owners need to treat as urgent, onboarding difficulties, slow connectivity | Clarify requirements with owners and improved access to TCK machines |
+| aarch64 Mac | JDK 17 | 6 | Bottlenecked on having only 1 machine for TCK | 2nd machine now added to temurin-compliance project |
+| aarch64 Mac | JDK 11 | 6 | Bottlenecked on having only 1 machine for TCK | 2nd machine now added to temurin-compliance project |
+| arm32 Linux | JDK 8 | 11 | Unclear, possibly slow to add an owner to it, also short-handed, team vacations | Close follow-up to ensure ownership early |
+| x86 Solaris | JDK 8 | 10 | Delayed on packaging issue | Issue now fixed, [temurin-build/3061](https://github.com/adoptium/temurin-build/issues/3061) |
+| sparcv9 Solaris | JDK 8 | 20 | Considered a best effort platform, team would/should prioritize this last | No specific action required, Reduction in effort on other platforms, and no respins would mean these lower priority platforms get released quicker |
```

Comment:
While it will always be a lower priority platform I think we should call out that our resources on this platform are very limited in quantity and performance. While that doesn't justify a 20 day delay it is worth calling it out, particularly given that some jobs were hitting out 50 hour timeouts so ended up being started from scratch, losing a couple of days at a time, and we can try and compensate for that by adjusting timeouts in the future. This has been true of the TC runs and the other AQA tests which have seen some machine differences (https://github.com/adoptium/infrastructure/issues/2729)